src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -47,6 +53,21 @@
     <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\AES\AesFactory.cs">
       <Link>CommonTest\AlgorithmImplementations\AES\AesFactory.cs</Link>
     </Compile>
+    <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\DSA\DSAFactory.cs" Condition="'$(TargetGroup)'=='netstandard1.7'">
```

Comment:
This is ok, but in general what we are trying to do in order to make things clearer, is to create two `ItemGroup`s, one where we add all of the tests that apply for both netcoreapp1.0 and 1.1, and one just specific to 1.1 like:
```xml
<ItemGroup Condition="'$(TargetGroup)'=='netstandard1.7'">
   <Compile Include="...
```